eng/common/TestResources/SubConfig-Helpers.ps1

```diff
@@ -207,6 +213,9 @@ function BuildAndSetSubscriptionConfig([string]$baseSubConfigJson, [string]$addi
     $subConfigs = $additionalSubConfigsJson | ConvertFrom-Json -AsHashtable
```

Comment:
We should probably also check these strings.

Reply:
I check them below for string in the loop. The outer type will always be array since we type check it in the pipeline yaml. I can add another string check here for safety but it will only guard against local input I think